### DISCUSSION TITLE
希望增加屏蔽动态充电专属视频功能

### DISCUSSION THREAD

**Discussion by 2SHANINGITHUB** [the1812/Bilibili-Evolved#4776] [功能建议 / Ideas]
充电专属视频经常在动态界面刷屏，又看不了，希望增加屏蔽功能

**Comment by the1812** [comment]
现成功能: `动态过滤器 - 移除充电专属动态`

**Reply by Geeyun-JY3** [reply]
我试过了，好像不行，充电视频的投稿动态还是在的。

**Reply by 2SHANINGITHUB** [reply]
试了，充电动态可以移除，充电视频无法移除

**Reply by Geeyun-JY3** [reply]
补截图，实测新旧版动态首页都不行（顺带一提，`动态过滤器`的`类型`部分选项在旧版动态首页不可见了，是失效了吗？）：
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/a3b1870e-edb7-4eaa-bcaa-8ac1e0697794)
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/54ccaf69-23d3-4060-8c16-8af1e79e3d02)

**Comment by hyb-store** [comment]
确实。而且首页这个动态弹窗，还不显示“充电专属”。每次点进去都看不了
![image](https://github.com/user-attachments/assets/6b62f07b-dde1-4338-9e58-7f435f1f75ca)
